receiver/tcpline.go

```diff
@@ -60,11 +59,11 @@ finish up. We should probably add a read-timeout in the future.
 func (tl *TCPLine) Start() error {
 	tcpip, err := net.ResolveTCPAddr("tcp", tl.Address)
 	if err != nil {
-		return skogul.Error{Source: "tcp receiver", Reason: fmt.Sprintf("unable to resolve address %s", tl.Address), Next: err}
```

Comment:
The default error message by golang here is: "lookup {tl.Address}: Temporary failure in name resolution", e.g. "lookup lolcalhost: Temporary failure in name resolution" (tl.Address = "lolcahost").

I think the current error message is a nice clarification to the user (e.g. in case of a typo), instead of a bit of a more cryptic (IMO) low-level error message.

I think that (example)
`return fmt.Errorf("unable to resolve address %s, err: %w", tl.Address, err)`
could be a nice, user-friendly error message.